L1Trigger/DemonstratorTools/src/codecs_htsums.cc

```diff
@@ -42,7 +43,7 @@ namespace l1t::demo::codecs {
       math::XYZTLorentzVector v(0, 0, 0, l1tmhtemu::MHT_t(x(l1tmhtemu::kMHTMSB, l1tmhtemu::kMHTLSB)).to_int());
       l1t::EtSum s(v,
                    l1t::EtSum::EtSumType::kMissingHt,
-                   l1tmhtemu::Et_t(x(l1tmhtemu::kHTMSB, l1tmhtemu::kHTLSB)).to_int(),
+                   l1tmhtemu::MHT_t(x(l1tmhtemu::kHTMSB, l1tmhtemu::kHTLSB)).to_int(),
```

Comment:
I'm not sure these produce the same values. `Et_t` can be set this way because it is an ap_int. `MHT_t` is an ap_fixed. Won't this only set the integer bits right now?